ckanext/odm_nav/templates/header.html

```diff
@@ -76,7 +77,7 @@
  <div class="debug">Controller : {{ c.controller }}<br/>Action : {{ c.action }}</div>
  {% endif %}
  {% endblock %}
- <div class="container">
```

Comment:
we need to keep this fixed width layout as long as we do not have a full responsive concept, it will will make it too restless and buggy. 
Will merge while keeping the container.